### ISSUE TITLE
Hypergeometric series of log 2

### ISSUE THREAD

**Issue by oovm** [Mysticial/y-cruncher-Formulas#10]
I would like to know if this type of formula can be supported by `y-cruncher`, I'm trying to find more similar formulas.

$$
\begin{aligned}
\log2
&=\frac{1}{108}\sum_{k=0}^{∞}\frac{P(k)}{Q(k)}\\
P(k)
&=1123+3680k+2800k^2\\
Q(k)
&=41472^k(6k+1)(6k+3)(6k+5)\binom{k-\frac{1}{2}}{4k}\\
&=(-1)^n\frac{162^k(6k+1)(6k+3)(6k+5)(2k)!(6k)!}{k!(3k)!(4k)!}
\end{aligned}
$$

```wolfram
P[n_]:=2800n^2+3680n+1123;
Q[n_]:=41472^n(1+6n)(3+6n)(5+6n)Binomial[n-1/2,4n];
N[Sum[P[k]/Q[k],{k,0,Infinity}]/108-Log[2],10000]
```

**Comment by Mysticial** [comment]
It looks hypergeometric. So it should be fine. Can't speak for its performance though.

**Comment by jzhansen** [comment]
Hi Alex,

Interesting log(2) series. It is new and belongs to a family that comes
from Lehmer's work in the 80's. It has a cost of 1.4314 and performs better
than the secondary 4 terms Machin like formula. I have put it in a custom
file providing a G3 version.

I am also sending a very fast formula for log(2) from T. Amderbeham in a G5
version (it has a cost of 1.9226) together with a new hypergeometric series
(the fastest one) that I have recently found. It has a cost of 1.3001 and
performs better than the primary 3-term Machin-like formula. I will also
send you a G2 or perhaps G3 custom files for this series later this week.

Your truly,

Jorge

**Comment by Mysticial** [comment]
As an update, I have another long email thread with Jorge.

There's going to be some new formulas for Log(2) and Log(3) coming in the next y-cruncher release along with the next batch of formulas.

Log(2) is getting quite a bit faster now and that is a big deal because it's such a fundamental constant and has implications for the Log-AGM algorithm and (to a lesser extent), the Euler-Mascheroni Constant.

A lot of stuff is being reworked behind the scenes in y-cruncher to be able to use these faster Log(2) formulas since it's been decades since the last movement on the best log(2) algorithm. That 3-term Machin formula has been hard-coded into a number of places in y-cruncher that need to be cleaned up. And that the log(x) function has also been hard-wired to Machin-like formulas that it's taking a lot of refactoring to decouple.

Basically a stirring the pot by changing such an old formula is exposing a lot of tech debt in y-cruncher.

**Comment by Mysticial** [comment]
Now that Jorge has [published his new formulas on MO](https://mathoverflow.net/questions/463436/are-these-fast-convergent-series-for-log2-log3-and-log5-already),

I've gone ahead and added them here: https://github.com/Mysticial/y-cruncher-Formulas/commit/4be450d8cd0e0328604ebe689a1f892fa2118422

They will all be in y-cruncher v0.8.4 and will show up automatically if you pick the relevant log(x) values.

**Comment by oovm** [comment]
I also asked about some formulas I found on the Chinese forum. 

Maybe the proof ideas are similar.

https://www.zhihu.com/question/641236019/answer/3377267073

**Comment by oovm** [comment]
It seems that the convergence speed is related to the exponential term, and the existing coefficients are beyond my improvement range.

**Comment by jzhansen** [comment]
Hi oovm,

Hi, my name is Jorge Zuniga. I checked out the zhihu social network link you left above. It is really interesting. 

A user named huang-de-ping-82  or 
Xiao Heng Ye (I guess)

[笑横野]

placed this [answer](https://www.zhihu.com/question/641236019/answer/3377267073) to an hypergeometric series proof for log(2) using a generating function and integral approach. 

It is very interesting because he generalized this proof for many series for log(2), log(3), Pi, log(1+b) and arctan(1/b). The basic formulas come from the hypergeometric motive [[1,1/2],[1/4,3/4]] or factorial ratio (2n)! (3n)! / n! / (4n)! that he extends to other ones more complex n! ((m+1)n)! (2mn)! /(mn)! / (2(m+1)n)! with m a positive integer.

I have seen similar formulas from professor Zhi Wei Sun [here](https://arxiv.org/pdf/2307.03086.pdf) Eqs. 1.44 and 1.45. for logarithms with different b values and m=1.

I have placed [here in MO](https://mathoverflow.net/questions/463436/are-these-fast-convergent-series-for-log2-log3-and-log5-already) some (I guess), new unproven series to compute log(2), log(3) and log(5) but they belong to a different family since the basic formulas come from the hypergeometric motive [[1,1/2],[1/6,5/6]] or factorial ratio (2n)!^2 (3n)! / n! / (6n)!.

I wonder if  [笑横野] could prove these (perhaps some generalized) series following a similar generating function approach. I have no way to contact him (her) in zhihu social network, perhaps you can tell him (her) that a proof of these expressions is needed.

Please feel free to give him (her) my e-mail jorge(dot)zuniga(dot)hansen(at)gmail(dot)com for further contact.

My Best Regards

Jorge
